spec/src/main/asciidoc/jakarta-ee.adoc

```diff
@@ -115,6 +117,11 @@ This fragment shows how the application might request injection of a `CarReposit
 
 This integration between CDI and Jakarta Data allows for seamless management of repository instances within Jakarta EE applications.
 
+==== CDI Events
```

Comment:
We should differentiate these lifecycle events from other events, such as Domain Event, see: https://github.com/jakartaee/data/issues/374

Reply:
Honestly I don't really see what the difference is. How is a "Domain Event" different from a lifecycle event for an entity that you're choosing call an "aggregate"?

Reply:
Technically, there is no difference.

Reply:
Right, that's what I thought.